### DISCUSSION TITLE
Provisioner responsible for Epic Games

### DISCUSSION THREAD

**Discussion by mmcdermo2** [actions/virtual-environments#4542] [Installed software]
We've just build out a VHD for Windows 2019 for use a private Azure DevOps Agent.  I've noticed however that "Epic Games" has been installed, despite not being listed in the installed software in the readme.  I'd like to remove it if possible as it's not something we're ever likely to use.

I've gone through the repo but I cannot figure what provisioner/install script is responsible for installing it.  The only thing I can find in the repo that mentions Epic Games is Idea #2003, which never got implemented.

Can anybody provide any pointers at all ?

**Comment by al-cheb** [comment]
Hey, @mmcdermo2 
It's part of Visual Studio - https://devblogs.microsoft.com/visualstudio/installing-the-unreal-engine-in-visual-studio/

> Once the initial Visual Studio installer has completed, the Epic Games Launcher will run and allow you acquire and install the actual Unreal Engine.

![image](https://user-images.githubusercontent.com/47745270/142607545-fefdb7e4-3e9a-4cec-97fc-1617743b8566.png)

![image](https://user-images.githubusercontent.com/47745270/142607778-55715c5c-0f1a-4b55-820a-eddcb9065590.png)

**Reply by mmcdermo2** [reply]
Hi @al-cheb 

Oh wow, sorry, never realized.  Thanks very much for letting me know.  Guess I won't be excluding it after all then :)

Thanks for the quick response
